x-pack/solutions/observability/plugins/synthetics/common/runtime_types/alert_rules/common.ts

```diff
@@ -44,11 +44,31 @@ export const AlertStatusMetaDataCodec = t.interface({
   }),
 });
 
+const StaleMetaDataCodec = t.partial({
+  isDeleted: t.boolean,
+  isLocationRemoved: t.boolean,
+});
+
 export const StaleAlertStatusMetaDataCodec = t.intersection([
   AlertStatusMetaDataCodec,
+  StaleMetaDataCodec,
+]);
+
+const MissingPingMonitorInfoCodec = t.intersection([
```

Comment:
this is unecessary, just a simple TS we should do, a codec is only required when we are validating user inputs